app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -708,6 +728,42 @@ class StatePlayerRecyclerViewAssembler private constructor(
     }
   }
 
+  private fun createBannerConfetti() {
+    val x = bannerConfettiView!!.width / 3
```

Comment:
Maybe pass the confetti view into this create method since you can guarantee at the call site it's not-null (which in turn lets you get rid of all of these ``!!``s).

Reply:
Done